# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1392283481 -3600
#      Thu Feb 13 10:24:41 2014 +0100
# Node ID 020c0fe578a7d044dc394d09ae49b60d3c84a502
# Parent  2bd78ab75b230e1cc319871f2cbddcfb42408120
Add missing file to pkg/module.mk.

scripts/pkg/module.mk: Add default_prefix.m to list of files.

diff --git a/scripts/pkg/module.mk b/scripts/pkg/module.mk
--- a/scripts/pkg/module.mk
+++ b/scripts/pkg/module.mk
@@ -1,15 +1,16 @@
 FCN_FILE_DIRS += pkg
 
 pkg_PRIVATE_FCN_FILES = \
   pkg/private/build.m \
   pkg/private/configure_make.m \
   pkg/private/copy_files.m \
   pkg/private/create_pkgadddel.m \
+  pkg/private/default_prefix.m \
   pkg/private/describe.m \
   pkg/private/dirempty.m \
   pkg/private/extract_pkg.m \
   pkg/private/finish_installation.m \
   pkg/private/fix_depends.m \
   pkg/private/fix_version.m \
   pkg/private/generate_lookfor_cache.m \
   pkg/private/get_description.m \
